### PR TITLE
Improve english text on welcome screen

### PR DESCRIPTION
**Pull request by n1313**
This PR improves english text on welcome screen. Russian text will be improved separately.

### CODE REVIEW

File: src/components/welcome.jsx [hunk 1]
```diff
@@ -2,44 +2,35 @@ import { Link } from 'react-router';
...
 
-    <p>
-      <b>No Ads</b>
-    </p>
+    <h4>No ads</h4>
 
     <p>
-      FreeFeed is a non-profit project. We do not show ads, we do not collect our users’ data, and
-      we do not sell anything to anyone.
+      FreeFeed is backed and maintained by a non-profit organisation. We do not show ads, we do not
```
Comment (dsumin) [inline src/components/welcome.jsx]: @n1313 organisation => organization

**Review by dsumin — COMMENTED** [review]
@n1313: Looking at the google document it looks like there was a plan to change `...sign in if you already have an account` to  `...sign in`

**Comment by n1313** [comment]
@dsumin исправлено, спасибо!

**Review by dsumin — APPROVED** [review]
LGTM

**Pull request merged**